social_core/tests/backends/test_twitch.py

```diff
@@ -1,5 +1,52 @@
 import json
 from .oauth import OAuth2Test
+from .open_id_connect import OpenIdConnectTestMixin
+
+
+class TwitchOpenIdConnectTest(OpenIdConnectTestMixin, OAuth2Test):
+    backend_path = 'social_core.backends.twitch.TwitchOpenIdConnect'
+    user_data_url = 'https://id.twitch.tv/oauth2/userinfo'
+    issuer = 'https://id.twitch.tv/oauth2'
+    expected_username = 'test_user1'
+    openid_config_body = json.dumps({
+        "authorization_endpoint": "https://id.twitch.tv/oauth2/authorize",
```

Comment:
Please use single quotes to follow existing code style.

Reply:
Ah, my mistake on that one. I normally use single quotes. I will fix that.